projects/extension/src/containers/Options.tsx

```diff
@@ -181,6 +227,146 @@ const Options: React.FunctionComponent = () => {
           </FormGroup>
         </FormControl>
       </TabPanel>
+      <TabPanel value={value} index={2}>
+        <Accordion
+          expanded={expanded === "panel1"}
+          onChange={handleAccordionChange("panel1")}
+        >
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon />}
+            aria-controls="panel1bh-content"
+            id="panel1bh-header"
+          >
+            <Typography className={classes.heading}>Error Logs</Typography>
+            <Typography className={classes.secondaryHeading}>
+              <Badge
+                badgeContent={errLogs.length}
+                showZero
+                color="error"
+              ></Badge>
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <div style={{ maxHeight: "500px", overflow: "auto" }}>
+              {errLogs.map((d: string) => {
+                const res = d.split("|")
+                return (
+                  <p style={{ lineHeight: "1.2rem" }}>
+                    <span
+                      style={{
+                        fontSize: "0.8rem",
+                        fontWeight: "bold",
+                      }}
+                    >
+                      {res[0]}
+                    </span>
+                    <span
+                      style={{
+                        fontSize: "0.8rem",
+                        fontStyle: "oblique",
+                        margin: "0 0.5rem",
+                      }}
+                    >
+                      {res[1]}
+                    </span>
+                    <span>{res[2]}</span>
+                  </p>
+                )
+              })}
+            </div>
+          </AccordionDetails>
+        </Accordion>
+        <Accordion
+          expanded={expanded === "panel2"}
+          onChange={handleAccordionChange("panel2")}
+        >
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon />}
+            aria-controls="panel2bh-content"
+            id="panel2bh-header"
+          >
+            <Typography className={classes.heading}>Warning Logs</Typography>
+            <Typography className={classes.secondaryHeading}>
+              <Badge
+                badgeContent={warnLogs.length}
+                showZero
+                color="primary"
+              ></Badge>
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <div style={{ maxHeight: "500px", overflow: "auto" }}>
+              {warnLogs.map((d: string) => {
+                const res = d.split("|")
```

Comment:
Also, I don't think this is a good idea. What if the `target` or the message contains a `|`?
I'd prefer if we just kept the date, log level, target, and message as individual fields in the `LogKeeper`, instead of concatenating them and splitting them later